jib-maven-plugin-extension-api/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+  id 'net.researchgate.release'
+  id 'maven-publish'
+}
+
+dependencies {
+  api project(':jib-build-plan')
+  api project(':jib-plugins-extension-common')
+  api "org.apache.maven:maven-core:${dependencyVersions.MAVEN_API}"
```

Comment:
I guess we expose maven core api in our api huh.

Reply:
The API method signature will be like `extend(ContainerBuildPlan, MavenProject, MavenSession, ...)`, so I guess we need to expose the maven-core, unless there's something else workable?